lib/matplotlib/animation.py

```diff
@@ -136,15 +172,13 @@ class MovieWriter(object):
     The use of the context manager ensures that setup and cleanup are
     performed as necessary.
 
-    frame_format: string
-        The format used in writing frame data, defaults to 'rgba'
     '''
 
     def __init__(self, fps=5, codec=None, bitrate=None, extra_args=None,
                  metadata=None):
         '''
-        Construct a new MovieWriter object.
-
+        Parameters
+        ----------
         fps: int
```

Comment:
Maybe change these to add the space before the colon that's needed while you're here, as done below.